LIPs/LIP-36.md

```diff
@@ -62,13 +65,13 @@ The following fields are added to the `EarningsPool.Data` struct:
 Add the following steps to the algorithm described in the [spec](https://github.com/livepeer/wiki/blob/master/spec/streamflow/spec.md#rewards):
 
 - Set `activeCumulativeRewards = cumulativeRewards`
-- Let `earningsPool` be orchestrator's earnings pool for the current round and `prevEarningsPool` be orchestrator's earnings pool for the previous round
+- Let `earningsPool` be orchestrator's earnings pool for the current round and `prevEarningsPool` be orchestrator's earnings pool for the previous round. 
+    - If `prevEarningsPool.cumulativeRewardFactor == 0` set it to the `cumulativeRewardFactor` of the `earningsPool` for the orchestrator's `lastRewardRound`
```

Comment:
A state update isn't actually necessary here right since the calculation can just reference the earningsPool for the lastRewardRound without updating prevEarningsPool?

Reply:
fixed in 4823679